lib/svtplay_dl/__init__.py

```diff
@@ -303,8 +303,8 @@ def options_subs_dl(subfixes):
     if options.merge_subtitle and not options.subtitle:
         options_subs_dl(subfixes)
 
-
-    if len(videos) == 0:
```

Comment:
why the change in logic?

Reply:
Its an alternative way of checking that 'len(videos) == 0'(length of array, list, tuple). This solution does the same as 'len(videos) == 0' but also include the case that  'videos is None'.

Reply:
I see, thanks

Reply:
Can `videos` ever be None? :)

Reply:
I guess not :)